src/App.js

```diff
@@ -14,13 +15,14 @@ function App() {
   const { getNextUrl, answers, innitiateNextSong } = usePlaylist(playlist)
 
   // useAudio hold and controll the audio element
-  const { setSongUrl, toggle, stop, isPlaying, duration } = useAudio()
+  const { setSongUrl, toggle, stop, isPlaying, duration, volume } = useAudio()
 
   const [newAnswers, setNewAnswers] = useState(answers)
 
   // //set a new song
   useEffect(() => {
-    !isPlaying && setSongUrl(getNextUrl)
+    if (isPlaying) return
+    setSongUrl(getNextUrl)
```

Comment:
I am not a fan of your early out. Looks a bit easy to misinterpret. 

```suggestion
    isPlaying || setSongUrl(getNextUrl)
```

Reply:
Oki doki, then i wrap the code inside the if statement.